front/src/reducers/mapViewer/__tests__/mapViewerReducer.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { createStoreWithoutMiddleware } from 'Store';
```

Comment:
:warning: Import is incorrect
```suggestion
import { createStoreWithoutMiddleware } from 'store';
```

Reply:
Done